torch/onnx/_internal/exporter.py

```diff
@@ -96,6 +115,16 @@ def resolve(value: Optional[T], fallback: Union[T, Callable[[], T]]) -> T:
 
         self.opset_version = resolve(options.opset_version, _DEFAULT_OPSET_VERSION)
         self.dynamic_shapes = resolve(options.dynamic_shapes, False)
+        # TODO: Prevent circular dep + [onnx]script dep
+        import torch.onnx._internal.fx.dynamo_graph_extractor as dynamo_graph_extractor
+        from torch.onnx._internal.fx import (  # TODO: onnxscript is not available on Pytorch CI yet
```

Comment:
We can maybe also delay the table evaluation?

Reply:
`ResolvedExportOptions` is only evaluated when an export has started. This is as delayed as it can be.
`decomposition_table` is not evaluated when we do `import torch` or `import torch.onnx`, etc.

Reply:
Sorry I meant the DEAFAULT_xxx table so as to avoid requiring onnxscript when importing these lines? Just a thought. May not be a good idea.

Reply:
Yeah, that is something I also feel needs another round of refactoring. We shouldn't have code being called on global scope as `decomp.py` does. 

This can be tackled along with the `passes` redesign, as decomposition is done as a pass today